.gitignore

```diff
@@ -36,7 +36,7 @@ data/
 *.jpeg
 *.jpg
 *.md
-*2.bib
+!Rmd/TDT2.bib
```

Comment:
why? TDT2.bib should not be edited or committed. It is generated with code from TDT.bib

Reply:
I added a name in the traitsstrap packaged, I thought this was done manually...
how do I remove this file from the commit list then?

Reply:
line 40 of 1_trait_driver_drake.R has a list of packages to cite. Ensures that the correct version is cited.